recipes/holoplot/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "holoplot" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "2b32b78bea003085413634cb83a8c2a5f48fe41a338056b5b15f5673798c1be4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - bokeh >=0.12.15
+    - holoviews >=1.10.4
+    - pandas
+
+test:
+  imports:
+    - holoplot
+    - holoplot.examples
+    - holoplot.examples.assets
+    - holoplot.examples.user_guide
+    - holoplot.examples.user_guide.images
+    - holoplot.tests
+  requires:
+    - coveralls
+    - flake8
+    - nbsmoke >=0.2.0
+    - nose
+    - parameterized
+    - pytest
```

Comment:
Also a little surprising that both `nose` and `pytest` are needed. Any thoughts on why that is?

Reply:
Looking at the source it doesn't seem to need `nose`.  It does list both of them in its `setup.py`

Reply:
The unit tests currently use nose (although the plan is probably to switch to pytest, but not straight away). Meanwhile, the notebook tests use pytest. 

Ideally, I think the unit tests and (quick) notebook tests should be run on conda-forge, as is done for other pyviz projects, e.g. https://github.com/conda-forge/datashader-feedstock/blob/master/recipe/meta.yaml#L42. 

Does that sound ok?

If so, would it be possible for me to make some modifications to this recipe here?

Reply:
Thanks for clarifying.

That seems fine. You could do a PR against this PR or we could save it for the feedstock.

Reply:
I made a PR for @CurtLH to evaluate (https://github.com/CurtLH/staged-recipes/pull/2).